application/features/password/password.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Password
+  In order to update the password of a specific user
+  As an authorized user
+  I need to be able to update a specific user password
+
+  Scenario: Provide a new password for an existing user
+    Given I receive an existing employee id
+      And the requestor is authorized
+      And the user does not already have a password
+      And I receive a password
+    When I receive a request to create a password for a specific user
+    Then a new password hash should be stored
+      And the last changed date should be stored as the instant it was stored
+      And the last changed date should be stored in UTC
+      And the last synched date should be stored as the instant it was stored
```

Comment:
Another option/considering is to sync the user anytime they change their password. That could provide a mechanism for forcing an individual sync.